enterprise/api_client/lms.py

```diff
@@ -233,6 +234,7 @@ def enroll_user_in_course(self, username, course_id, mode, cohort=None):
                 'is_active': True,
                 'mode': mode,
                 'cohort': cohort,
+                'enterprise_uuid': enterprise_uuid
```

Comment:
[mild suggestion] You could cast the `enterprise_uuid` to a `str()` on this line.  Then the caller doesn't have to worry about passing in a UUID object vs. a string, since the `post()` call will "do the right thing".

Reply:
@iloveagent57  It's done